pytorch_pfn_extras/onnx/strip_large_tensor.py

```diff
@@ -1,43 +1,41 @@
-# mypy: ignore-errors
-
 import argparse
 import json
 import operator
```

Comment:
To type onnx.GraphProto/ModelProto this will also be needed:
```suggestion
import onnx
import operator
```